src/jenkins/common/Openenclave.groovy

```diff
@@ -120,4 +120,32 @@ def emailJobStatus(String status) {
     )
 }
 
-return this
\ No newline at end of file
+def WinCompilePackgeTest(String dirName, String buildType, String hasQuoteProvider, Integer timeoutSeconds) {
```

Comment:
I agree this is a good thing to add. I don't quite understand the name though. Is this function supposed to "compile package & run tests". At the very least a doxygen-style comment for the function would be good.

Reply:
Comment added.